new-log-viewer/src/utils/downloadDecompressedLogs.ts

```diff
@@ -0,0 +1,23 @@
+interface downloadDecompressedLogsProps {
+    blob: Blob,
+    fileName: string,
+}
+
+/**
+ *
+ * @param blob.blob
+ * @param blob
+ * @param fileName
+ * @param blob.fileName
+ */
+const downloadDecompressedLogs = ({blob, fileName}: downloadDecompressedLogsProps) => {
+    const url = URL.createObjectURL(blob);
+    const link = document.createElement("a");
+    link.href = url;
+    link.download = `${fileName}-exported-${new Date().toISOString()
+        .replace(/[:.]/g, "-")}.log`;
```

Comment:
Shall we generate the exported file name outside of this function to make this more generic? e.g. 
```suggestion
    link.download = fileName;
```